### DISCUSSION TITLE
Remote Test RT - Guide Help Request

### DISCUSSION THREAD

**Discussion by socelillyhammer** [GerryDazoo/Slinger#388] [Q&A]
Hi All,
First Id like to thank everyone for the contributions and help for what we all have found and shared, including @GerryDazoo his work.

I was hoping I can get some help on the remote.  As I am not so code savvy. 

In reading the attached release notes there is supposed to be a RT remote test button which I am not seeing on my remote.  There seems to be a way to change the IR code so that it can work with the STB. But I am lost.

If anyone can help get me going I'd appreciate it.

Here is the release notes which discuss the RT button
[V3.0x_ReleaseNotes.pdf](https://github.com/GerryDazoo/Slinger/files/10127209/V3.0x_ReleaseNotes.pdf)

**Comment by socelillyhammer** [comment]
Need some guidance on the IR receiver for the 500.

I've compared the remote map logs #165 against the default remote for verizon fios.  The code numbers match up.  So I know thats not my issue.

I'm wondering if there is a way to test my IR receiver or if its bricked or the settings before SB went dark are stuck on a device that doesnt work with my STB.  And if the last one is the case, can it be fixed? or would a hard reset work (now that I have my ID and password)

**Reply by sp222222** [reply]
again - 1) you already have a discussion thread open on this very issue. 2) did you or did you not hard reset the 500 you have? If you have hard reset it, this is a moot point in attempting to get a remote working with this software. You would need an external "b remote" $30. (I have already explained this to you once).
2) thread 165# has nothing to do with your issue. nothing. nothing.............
3) pdf refers to 307 so grab all the files needed from that version of slinger. the basic remote html has the RT field in which if you put ?39 (for example) it will hopefully respond on screen after 5-7 secs with a key code that coorsponds with a remote button on actual remote.  the log output gives you all the key codes that your slingbox has loaded into it for that input. if it shows no key codes (in brackets, lots of two digit numbers), then input was not configured on the slingbox itself.
what the rest of us have done the hard way is take each keycode and put a button on the remote.txt to match what it does. but only by using ?XX where XX=keycode number will that be known.

Good luck.

**Reply by socelillyhammer** [reply]
Apologies for not understanding the rules of how these threads work as this thread was for questioning how to test the IR receiver.  As an attorney who is not educated in coding and trying to thumb through all this material to get my SB up and running, its quite difficult in deciphering this language.  Hecne the reason why I offered to pay anyone who can set up and get me going on both my SBs so i can remotely connect into them.

No i did not reset my box.  Even going back into the box with the SB remote to try and configure, my original remote and tv hdmi settings are still there so i know the box didnt reset.

as for thread 165, what do i know from it.  Again hence the reason to pay someone to get me on my way rather than waste time.

I have tried to look for version 307 but I dont know where to find it. again will scour through the message boards to see what i can find.

If you want, I have no problem paying you to get me on my way $500 a box.

**Reply by Toei79** [reply]
seriously man, attorneys have an amazing memory i bet you can learn this fast. 

i see this kinda like trial and error, more you see more familiar you get with it, but yea it can be complicated, its take me a lot of time  to barely understand this.

but whats your issue? im guessing its just the remote so far right? other stuff its already done .

anyway looks how moves all here its matter of time all bases get filled.

**Reply by socelillyhammer** [reply]
@sp222222 still cant find the 307 version.  so for a lousy $30 can you recommend an external b remote from amazon please and I assume I would have to code it in or it would just work?

Thank you

**Reply by GerryDazoo** [reply]
You do not need V3.07 with the RT command. I took it out because it no long had any value once we figured out the number was the same as the VideoSource.
As sp22222 said put a ? followed by the keycode you want to test into the "Channel" field and press enter
ie. ?47 will send keycode 47 to the slingbox and it will send the IR command to your device depending on what remote has been previously configured for your selected VideoSource.

**Reply by socelillyhammer** [reply]
@GerryDazoo @sp222222  Thank you both.  I appreciate the explanation and history of the versions.  In reading other threads, notes, etc. after many hours of research I have learned the history of the different versions and where it developed into today.  As we are used to a typical plug and play make everything work society, I've seen that coding is different - hence the importance of understand this history of problems and solutions thereafter - hence the coding. 

Again, thank you both for your time.  I will try the IR on my other slingbox today.  But in the interest of time and money  I have to see about this $30 external remote (which I hope will be a plug and play)

I would presume this would work? 

https://www.amazon.com/BlastIR-Wireless-Emitter-Control-Extender/dp/B074CPBDXV/ref=sr_1_14?c=ts&keywords=Remote+Control+Extenders&qid=1669918727&s=tv&sr=1-14&ts_id=11039361

**Reply by socelillyhammer** [reply]
In reading other posts on other forums, regarding the SB 500 - it seems the code does not work for it and whatever SB did screwed with the IR coding of the hardware.  

Unless I am a wrong and somene has the IR receiver working on the SB 500.

That being said, can someone please tell me if this is the correct external IR $30 piece I need to use to try and change channels?

https://www.amazon.com/BlastIR-Wireless-Emitter-Control-Extender/dp/B074CPBDXV/ref=sr_1_14?c=ts&keywords=Remote+Control+Extenders&qid=1669918727&s=tv&sr=1-14&ts_id=11039361

**Reply by GerryDazoo** [reply]
That is 100% wrong. If you haven't done a factory reset on your box, IR works as expected.

**Comment by sp222222** [comment]
thread 398# socelilly has your cure

**Comment by socelillyhammer** [comment]
Hi all.

Just want to thank you all for your help on the remote issue.

Ive tested a few and I found this to be the best.  It can work while youre not connected to your home wifi.  Ive also spoofed my GPS and IP on my phone to relocate me out of the country to test the functionality off the WiFi and it works perfectly.

So if you need a remote IR receiver, I would recomment this one

https://www.amazon.com/dp/B07ZSF46BX?psc=1&ref=ppx_yo2ov_dt_b_product_details
